opm/autodiff/BlackoilModelParametersEbos.hpp

```diff
@@ -72,7 +72,7 @@ SET_BOOL_PROP(FlowModelParameters, UpdateEquationsScaling, false);
 SET_BOOL_PROP(FlowModelParameters, UseUpdateStabilization, true);
 SET_BOOL_PROP(FlowModelParameters, MatrixAddWellContributions, false);
 SET_SCALAR_PROP(FlowModelParameters, TolerancePressureMsWells, 0.01 *1e5);
-SET_SCALAR_PROP(FlowModelParameters, MaxPressureChangeMsWells, 2.0 *1e5);
```

Comment:
@alfbr , @totto82 , it is mostly likely this change makes some cases fail.

Reply:
By changing it to `2.e5`, the running is recovered, while some slightly different iteration process was shown. One more testing is ongoing and it will be reported back.

Reply:
Did not figure out why the iteration behavior changed compared with master branch. But with resetting it back to `2.e5`, the running of the obtained problematic case is recovered.

Reply:
I hope #1855 will fix the problem. Testing in progress......

Reply:
#1855 fixed the problem.

Reply:
Good! What is the actual problem? Some exceptions we could not catch?

Reply:
With #1855 all exceptions should be caught. I have added a partial fix for this in this PR. I will remove it when #1855 is merged.

Reply:
Good! How the performance looks like? similar?

Reply:
I am testing it at the moment. I.e. The same case with `2e5`

Reply:
The 1e6 speeds up the computations due to less convergence issues.

Reply:
That is good to know. Is this the one problematic case we were trying to fix?  Can you post the performance information here so that I do not need to run it.